docs/spec.md

```diff
@@ -3101,8 +3124,13 @@ separate outputs to improve clarity
 #### Semantics
 
 Fills an `output` tensor with values in increasing order starting from zero
-along the `iota_dimension` dimension. More formally, `output[result_index] =
-constant(result_index[iota_dimension], element_type(output))`.
+along the `iota_dimension` dimension. More formally,
+
+* For non-quantized case: `output[result_index] = constant(
+        result_index[iota_dimension], element_type(output))`.
+* For quantized case: `output[result_index] = constant(
+        quantize(result_index[iota_dimension], element_type(output)),
+        element_type(output))`.
```

Comment:
Let's shorten this using the `is_quantized(output) ? quantize(foo, element_type(output)) : foo` notation from elsewhere in the spec. I like it better than my original proposal.

Reply:
done.